includes/Modules/Reader_Revenue_Manager.php

```diff
@@ -80,6 +90,44 @@ public function setup_services( Google_Site_Kit_Client $client ) {
 		);
 	}
 
+	/**
+	 * Checks whether the module is connected.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @return bool True if module is connected, false otherwise.
+	 */
+	public function is_connected() {
+		$options = $this->get_settings()->get();
+
+		if ( ! empty( $options['publicationID'] ) ) {
+			return true;
+		}
+
+		return false;
+	}
+
+	/**
+	 * Sets up the module's settings instance.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @return Module_Settings
+	 */
+	protected function setup_settings() {
+		return new Settings( $this->options );
+	}
+
+	/**
+	 * Cleans up when the module is deactivated.
+	 *
+	 * @since n.e.x.t
+	 */
+	public function on_deactivation() {
+		$this->get_settings()->delete();
+		$this->reset_data_available();
```

Comment:
The Reader Revenue Manager module, as it currently stands, does not have any data to work with, hence, it should neither use the `Module_With_Data_Available_State_Trait` trait, nor call the `reset_data_available` function.

Is there a specific reason this was added? If not, I'm happy to remove this as part of one of my upcoming PRs.

Reply:
Good spot @nfmohit, I missed this in CR. I believe this should indeed be removed, if you can do it via one of your forthcoming PRs that would be great.